### DISCUSSION TITLE
Unable to use the Multi select inside Alert-Dialog or Dialog component

### DISCUSSION THREAD

**Discussion by jawahar273** [sersavan/shadcn-multi-select-component#20] [Q&A]
Hi, I was planning to add multi-select on dialog component but I was not able to select item from the dropdown. Please suggest any alternative workaround for this issue.

https://github.com/user-attachments/assets/9f50ecb7-c7b4-4c8c-972a-bcbac0fb5d51

**Comment by olawalle** [comment]
add the tailwind class `pointer-events-auto` to the `PopoverContent` element within the multi-select file. solves the issue!

`<PopoverContent
          className="w-auto p-0 pointer-events-auto"
          align="start"
          onEscapeKeyDown={() => setIsPopoverOpen(false)}
        >` ....

**Reply by jmpena2011** [reply]
This fixes the problem for the select elements, but the "CommandInput" still has the problem. Any solution for the "CommandInput"?

**Comment by jmpena2011** [comment]
[See this comment](https://github.com/shadcn-ui/ui/issues/2139#issuecomment-2567021626): its works for me!
